.circleci/config.yml

```diff
@@ -295,15 +304,20 @@ commands:
           command: |
             sleep 30
             curl http://127.0.0.1:4040/api/tunnels
+      - run:
+          name: "ngrok: create twiML app with ngrok public_url"
+          command: |
+            cd server/scripts
+            node createTwimlApp.js
 
   configure-git-user:
     description: "Configures the git user for committing"
     steps:
       - run:
-         name: Git User Config
-         command: |
-           git config --global user.email "${GITHUB_EMAIL_SDK_BUILD}"
-           git config --global user.name "${GITHUB_USERNAME_SDK_BUILD}"
+          name: Git User Config
+          command: |
+            git config --global user.email "${GITHUB_EMAIL_SDK_BUILD}"
+            git config --global user.name "${GITHUB_USERNAME_SDK_BUILD}"
```

Comment:
Leading whitespaces

Reply:
@charliesantos I think this leading whitespace is correct? Without it the `name` would align under the `u` of `run` instead of under the `n` like other lines.

Reply:
Plus one to @kpchoy's reason.

Reply:
👍